### PR TITLE
Emulator api 28

### PR DESCRIPTION
**Pull request by Dartlexx**
This branch should be our mainline for now - API28 and API29 will be our future emulators

### CODE REVIEW

**Comment by Malinskiy** [comment]
@Dartlexx no peer review?

File: README.md [hunk 1]
```diff
@@ -16,14 +16,14 @@ docker-emulator-android is one of the components of [android-farm](https://githu
 # Usage
 For example to run default emulator options with Marshmallow (API 23):
 ```console
-$ docker run --rm --privileged -e ANDROID_ARCH="x86" -v /dev/kvm:/dev/kvm agoda-com/docker-emulator-android-23:latest
+$ docker run --rm --privileged -e ANDROID_ARCH="x86_64" -v /dev/kvm:/dev/kvm agoda-com/docker-emulator-android-23:latest
```
Comment (Malinskiy) [inline README.md:19]: x86 is significantly faster than x86_64. Please provide benchmarks if that's not the case anymore

File: templates/Dockerfile [hunk 2]
```diff
@@ -19,40 +19,44 @@ RUN dpkg --add-architecture i386 && \
...
     # Install Android SDK
     curl -L $LINK_ANDROID_SDK > /tmp/android-sdk-linux.zip && \
     unzip -q /tmp/android-sdk-linux.zip -d /opt/android-sdk-linux/ && \
     rm /tmp/android-sdk-linux.zip && \
     # Customized steps per specific platform
     yes | sdkmanager --no_https --licenses && \
-    yes | sdkmanager emulator tools platform-tools "platforms;{{ platform }}" "system-images;{{ platform }};google_apis;x86" --verbose | uniq && \
-    echo no | avdmanager create avd -n "x86" --package "system-images;{{ platform }};google_apis;x86" --tag google_apis && \
+    yes | sdkmanager emulator tools platform-tools "platforms;{{ platform }}" "system-images;{{ platform }};google_apis;x86_64" --verbose | uniq && \
+    echo no | avdmanager create avd -n "Pixel2" --package "system-images;{{ platform }};google_apis;x86_64" --tag google_apis && \
```
Comment (Malinskiy) [inline templates/Dockerfile:37]: Changing the name of the emulator will break other setups if they do some communication with the avd by name

File: templates/Dockerfile [hunk 3]
```diff
@@ -19,40 +19,44 @@ RUN dpkg --add-architecture i386 && \
...
-    (qemu-img convert -O qcow2 -c /opt/android-sdk-linux/system-images/{{ platform }}/google_apis/x86/vendor.img /opt/android-sdk-linux/system-images/{{ platform }}/google_apis/x86/vendor.qcow2 && \
-    mv /opt/android-sdk-linux/system-images/{{ platform }}/google_apis/x86/vendor.qcow2 /opt/android-sdk-linux/system-images/{{ platform }}/google_apis/x86/vendor.img || true) && \
     # Clean up
     apt-get -yq autoremove && \
     apt-get clean && \
     apt-get autoclean && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-COPY config.ini /root/.android/avd/x86.avd/config.ini
+# Convert large partitions to qcow2 to save space
```
Comment (Malinskiy) [inline templates/Dockerfile:47]: This will increase the docker image size by gigabytes. Please provide the comparison if that's not the case

**Comment by Dartlexx** [comment]
@Malinskiy Actually it is my mistake. I was treating this project as Agoda only. But it seems to be used someone else. So I will fix my error - I will rollback that change and create a branch for our own usage. I will also make 1 review to be required, you are right.

File: README.md (hunk 1 above)
Comment (Dartlexx) [reply README.md:19]: With the recent image, I haven't found that x86 is any faster than x86_64. But that is true for our current setup.

File: templates/Dockerfile (hunk 2 above)
Comment (Dartlexx) [reply templates/Dockerfile:37]: Yes, you are right. Thanks

File: templates/Dockerfile (hunk 3 above)
Comment (Dartlexx) [reply templates/Dockerfile:47]: Yes, it is twice bigger.

**Pull request merged**